catalog/app/containers/Bucket/requests/requestsUntyped.js

```diff
@@ -1102,7 +1113,9 @@ export async function resolvePackageRevision({ s3, bucket, name, revision }) {
     }
   } else if (TIMESTAMP_RE.test(revision) || revision === 'latest') {
     try {
-      return loadRevisionHash({ s3, bucket, name, id: revision })
+      // we need to await the result to catch the errors
+      // eslint-disable-next-line @typescript-eslint/return-await
```

Comment:
Or help Typescript recognize `loadRevisionHash` as `async`:

```js
export const loadRevisionHash = async ({ s3, bucket, name, id }) =>
  s3
    .getObject({ Bucket: bucket, Key: getRevisionKeyFromId(name, id) })
    .promise()
    .then((res) => ({
      modified: res.LastModified,
      hash: res.Body.toString('utf-8'),
    }))
```

Reply:
oh that worked thx